### ISSUE TITLE
Crashing when clicking "Home"

### ISSUE THREAD

**Issue by BarbourSmith** [MaslowCNC/GroundControl#201]
I've seen an issue and heard it mentioned in the forums that clicking the **Home** button on this week's build can cause GC to crash. I've had a hard time making the problem occur repeatably in a way that I can track it down, but here are the facts that I have found so far.

1) It seems to happen only in the compiled version which means that the error message flashes on the screen too fast for me to read what it says.
2) Opening the settings and changing the z-axis from enabled to disabled and back again seems to fix the issue?

Has anyone else seen this problem? Does anyone have any idea what the error message says when the program crashes?

**Comment by jbarchuk** [comment]
> the error message flashes on the screen too fast

Set up a video camera to record the error. Or screencap software, but that -might- introduce timing that causes the problem to -not- happen.

**Comment by BarbourSmith** [comment]
I tried the video camera idea, but I still couldn't quite get it. I think the issue was that my phone's frame rate is too slow. Then I realized that I could just put a pause command in the .bat file which launches the program so after it crashed it would hold the error open. Here's the message, an easy fix.
![error message](https://cloud.githubusercontent.com/assets/9359447/25096520/6365bf0c-2355-11e7-92dd-c2678829def8.JPG)
